### PR TITLE
feat(AWS Lambda): Add support for consumer group ID to kafka event

### PR DESCRIPTION
**Pull request by Phil-Pinkowski**
Context: this has just been added to the event trigger as a feature and is an extremely useful missing feature: https://aws.amazon.com/about-aws/whats-new/2022/08/aws-lambda-consumer-group-ids-msk-kafka-event-sources/

Closes: (No issue created yet for this)

### CODE REVIEW

**Comment by mnapoli** [comment]
Hi, just a heads up: unfortunately at the time we will not have the capacity to provide guidance or review a PR implementing that feature. Sorry that I'm not able to help more with this!

**Comment by awitterackspace** [comment]
+1 for this feature.

@mnapoli thanks for the update; any idea on when you'll have capacity to add this feature?

**Comment by simonarcher99** [comment]
+1 for this issue as well

File: lib/plugins/aws/package/compile/events/kafka.js
Comment (pgrzesik) [inline lib/plugins/aws/package/compile/events/kafka.js:244]: This shouldn't be here - there's a separate event for MSK under https://github.com/serverless/serverless/blob/main/lib/plugins/aws/package/compile/events/msk/index.js

**Review by pgrzesik — CHANGES_REQUESTED** [review]
Looks good overall, it would be great to add it to `msk` event as well for completeness - in the current form it will only work for self-manager Kafka clusters

File: docs/providers/aws/events/kafka.md
Comment (pgrzesik) [inline docs/providers/aws/events/kafka.md:26]: Would be also great to add references to https://github.com/serverless/serverless/blob/main/docs/providers/aws/guide/serverless.yml.md

**Comment by king-freshket** [comment]
+1 for this feature

File: lib/plugins/aws/package/compile/events/kafka.js
Comment (Phil-Pinkowski) [reply lib/plugins/aws/package/compile/events/kafka.js:244]: Ah I hadn't realised there was a separate MSK event, will move it here 👍

**Review by pgrzesik — APPROVED** [review]
Looks great, @Phil-Pinkowski 👍 It will probably need another review from @mnapoli or @medikoo before merging though.

**Review by medikoo — APPROVED** [review]
Thank you @Phil-Pinkowski ! That looks great

Great thanks @pgrzesik for help in pushing it forward

**Pull request merged**
